census/README.md

```diff
@@ -4,34 +4,40 @@ Predict a person's income level.
 
```

Comment:
While you are at it can you fix this as well. The README has two titles. 

Predicting Income with the Census Income Dataset

Predict a person's income level.